index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>BusMall</title>
+    <link href="style.css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Lato|Lora|Oleo+Script+Swash+Caps" rel="stylesheet">
+  </head>
+  <body>
+    <div class="wrapper">
+      <header>
+          <img class="logoPic" src="pictures/busMallLogo.png"/>
+          <h1>BusMall</h1>
+
+        <nav>
+          <ul>
+            <li><button type="button">Home</button></li>
+            <li><button type="button">Products</button></li>
+            <li><button type="button">About</button></li>
+            <li><button type="button">Contact</button></li>
+          </ul>
+        </nav>
+      </header>
+        <aside>
+          <h3>Results</h3>
+        </aside>
+        <div class="productContainer">
+          <div class="productWrapper" >
+            <img class="productImage" src="pictures/backwardsWateringCan.jpg"/>
+            <button class="selecter" type="button">Select</button>
+          </div>
+          <div class="productWrapper" >
+            <img class="productImage" src="pictures/iceCreamLock.jpg"/>
+            <button class="selecter" type="button">Select</button>
+          </div>
+          <div class="productWrapper" >
+            <img class="productImage" src="pictures/pizzaPouch.jpg"/>
+            <button class="selecter" type="button">Select</button>
+          </div>
+          <button class="vote" type="button">Vote</button>
```

Comment:
Don't forget a `<form>` when you have a Submit-type behavior. Not strictly necessary when we're wiring up all the behavior with JS, but if we were building this for real it would be sending the vote data _somewhere_.